source/opt/const_folding_rules.h

```diff
@@ -22,7 +22,7 @@
 #include "folding_rules.h"
 #include "ir_builder.h"
 #include "ir_context.h"
-#include "spirv/1.2/spirv.h"
+#include "latest_version_spirv_header.h"
```

Comment:
This header should be included before other local headers. Please also add blank line, so that clang-format doesn't reorder.

Reply:
Same